course_info/static/course_info/js/controllers/DetailsController.js

```diff
@@ -0,0 +1,110 @@
+                dc.courseInstance = dc.getFormattedCourseInstance(courseResult.data, membersResult.data)
+            } else {
+                $log.error(' CourseInstance record mismatch for id :'
+                    + dc.courseInstanceId + ',  fetched record for :' + courseResult.data.id);
+            }
+        };
+
+        dc.setCourseInstance = function (id) {
+
+            var course_url = djangoUrl.reverse(
+                'icommons_rest_api_proxy',
+                ['api/course/v2/course_instances/' + id + '/']);
+
+            var members_url = djangoUrl.reverse(
+                'icommons_rest_api_proxy',
+                ['api/course/v2/course_instances/'
+                + id + '/people/']);
+
+            var coursePromise = $http.get(course_url)
+                .error(dc.handleAjaxError);
+
+            var membersPromise = $http.get(members_url)
+                .error(dc.handleAjaxError);
+
+            $q.all([coursePromise, membersPromise])
+                .then(dc.handleLookupResults);
+        };
+
+        dc.stripQuotes = function(str){
+            // soem fields are coming over with quotes around them and those quotes
+            // are being displayed in the html.
+            // This strips off double quotes from the begining and ending of fields
+            return str ? str.trim().replace(new RegExp("^\"|\"$", "g"), "") : '';
+        };
+
+        dc.getFormattedCourseInstance = function (ci, members) {
+            // This is a helper function that formats the CourseInstance metadata
+            // and is combination of existing logic in
+            // Searchcontroller.courseInstanceToTable and Searchcontroller cell
+            // render functions.
+            courseInstance = {};
+            if (ci) {
+                courseInstance['title'] = dc.stripQuotes(ci.title);
+                courseInstance['school'] = ci.course ?
+                    ci.course.school_id.toUpperCase() : '';
+                courseInstance['term'] = ci.term ? ci.term.display_name : '';
+                courseInstance['year'] = ci.term ? ci.term.academic_year : '';
+                courseInstance['departments'] = ci.course.departments ? ci.course.departments : [];
+                courseInstance['course_groups'] = ci.course.course_groups ? ci.course.course_groups : [];
+                courseInstance['cid'] = ci.course_instance_id;
+                courseInstance['registrar_code_display'] = ci.course ?
+                ci.course.registrar_code_display +
+                ' (' + ci.course.course_id + ')'.trim() : '';
+                courseInstance['description'] = dc.stripQuotes(ci.description);
+                courseInstance['short_title'] = dc.stripQuotes(ci.short_title);
+                courseInstance['sub_title'] = ci.sub_title ? dc.stripQuotes(ci.sub_title) : '';
+                courseInstance['meeting_time'] = ci.meeting_time ? ci.meeting_time : '';
+                courseInstance['location'] = ci.location;
```

Comment:
For other fields, we're explicitly checking for a falsey value and using `''` in its place.  Any reason not to stick with that pattern here?  For the record, I'd prefer just putting the values in as is, but either way, I think we should pick one and go with it.